Project/Assets/_Scripts/_new_Scripts/Test/AnimationConfigurator.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AnimationConfigurator : MonoBehaviour
+{
+    Animator animator;
+    Camera cam;
+    
+    float y;
+    float targetY;
+    float remainingY;
+    bool isFacingRight;
+    float rotationSpeed;
+    float deltaTime;
+    Vector3 headDir;
+    Vector3 headPosition;
+    float headAngle;
+
+    void Start()
+    {
+        animator = GetComponentInChildren<Animator>();
+        cam = GameObject.Find("Camera").GetComponent<Camera>();
+        y = 0.0f;
+        rotationSpeed = 700.0f;
+    }
+
+
+    void Update()
+    {
+        deltaTime = Time.deltaTime;
+        isFacingRight = animator.GetBool("isFacingRight");
+        headPosition = cam.WorldToScreenPoint(animator.GetBoneTransform(HumanBodyBones.Head).position);
+        headDir = cam.WorldToScreenPoint(headPosition - Input.mousePosition);
+        headAngle = Vector3.Angle(headDir, transform.up * -1.0f);
+
+        if (Input.mousePosition.x - 20.0f >= headPosition.x)

```

Comment:
Chciałbym to co jest w Ifie żeby było prywatną funkcją z jakąś ładną nazwą. Bo nie wiem do czego ona służy, to samo  w kolejnych ifach